### PR TITLE
Fix yaml lint errors and warnings

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
Fixes: #387

Signed-off-by: Puneet Punamiya <ppunamiy@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [x] Yaml file complies with [yamllint](https://github.com/adrienverge/yamllint) rules.

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: wget/wget.yaml [hunk 1]
```diff
@@ -4,31 +4,37 @@ metadata:
...
-     description: The folder where we write the wget'ed file to  
+    - name: wget-workspace
+      description: The folder where we write the wget'ed file to
   steps:
     - name: wget
       image: "$(params.wget-option)"
       command: [wget]
-      args: ["$(params.options[*])", "$(params.url)", "$(params.diroptions[*])", "$(workspaces.wget-workspace.path)/$(params.filename)"]
+      args:
+        [
```
Comment (sthaha) [inline wget/wget.yaml:35]: this change isn't needed and I think it should be 

```yaml
args: [
  "$(params.options[*])", 
  "$(params.url)",
]
```

File: curl/curl.yaml [hunk 2]
```diff
@@ -4,17 +4,17 @@ metadata:
...
-     type: string
-   - name: options
-     description: options of url
-     type: array
-     default: []
-   - name: curl-image
-     description: option of curl image
-     type: string
-     default: "curlimages/curl"
+    - name: url
```
Comment (sthaha) [inline curl/curl.yaml:7]: this change isn't strictly required and changes the convention use throughout the catalog. could we please revert this whitespace style change 🙏 ?

**Comment by chmouel** [comment]
/lgtm

Looks good thanks! 

Would you like to give a try to add back the yamllint job on PR ? @vdemeester or @afrittoli may be able to help you..

**Comment by PuneetPunamiya** [comment]
> /lgtm
> 
> Looks good thanks!
> 
> Would you like to give a try to add back the yamllint job on PR ? @vdemeester or @afrittoli may be able to help you on where to do the change..

Yes sure but how do I need to do ?

**Comment by vdemeester** [comment]
> Yes sure but how do I need to do ?

@PuneetPunamiya you could apply the following diff (in a separate commit :wink: )

```diff
diff --git a/test/presubmit-tests.sh b/test/presubmit-tests.sh
index 005c2e8..57f7098 100755
--- a/test/presubmit-tests.sh
+++ b/test/presubmit-tests.sh
@@ -25,7 +25,6 @@
 # in a net-negative contributor experience.
 export DISABLE_MD_LINTING=1
 export DISABLE_MD_LINK_CHECK=1
-export DISABLE_YAML_LINTING=1
 
 source $(dirname $0)/../vendor/github.com/tektoncd/plumbing/scripts/presubmit-tests.sh
```

**Comment by vdemeester** [comment]
Well I think it works :upside_down_face:

**Pull request merged**